### ISSUE TITLE
Supervisor errors

### ISSUE THREAD

**Issue by Salvora** [allenporter/stream-addons#47]
Hello,
I get bunch of these supervisor errors when the addon is running.
supervisor-2022.01.1
core-2022.2.9
Home Assistant OS 7.4
RPI 4

```
22-02-27 19:00:29 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-46249' coro=<Scheduler._run_task.<locals>._wrap_task() done, defined at /usr/src/supervisor/supervisor/misc/scheduler.py:58> exception=AttributeError("'NoneType' object has no attribute 'get'")>
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/misc/scheduler.py", line 62, in _wrap_task
    await task.coro_callback()
  File "/usr/src/supervisor/supervisor/misc/tasks.py", line 417, in _watchdog_addon_application
    if addon.in_progress or await addon.watchdog_application():
  File "/usr/src/supervisor/supervisor/addons/addon.py", line 473, in watchdog_application
    port = self.ports.get(f"{t_port}/tcp", t_port)
AttributeError: 'NoneType' object has no attribute 'get'
```

**Comment by allenporter** [comment]
Hi, I'm not able to reproduce this. Would you mind including more steps for how this happens?

**Comment by Salvora** [comment]
Umm,
I just saw this error and tried disabling all the addons 1 by 1. After disabling the **RTSPtoWeb - WebRTC**, the error is gone.
To replicate,
1- Start the addon
2- Enable "Start on boot" and "Watchdog"
3- Check that the **RTSPtoWebRTC Integration** is running.
4- Restart HA
5- At this point after restart, The integration always fails to load for me. and I get these supervisor errors.

**Comment by originalbase** [comment]
core-2022.3.1
supervisor-2022.01.1
Home Assistant OS 7.4
RPi 4

Repository: https://github.com/allenporter/stream-addons
Addon: RTSPtoWeb - WebRTC 1.2.1

Addon ist starting, Integration loads, no problems after restart
BUT the supervisor log is flooded with the message mentioned above (every 2 minutes).

**Comment by tammeryousef1006** [comment]
i found temp solution for this issue by turn off watchdog for the addon

**Comment by tammeryousef1006** [comment]
i found temp solution for this issue by turn off watchdog for the addon

**Comment by TekFan** [comment]
Same issue here.
Disabling watchdog fixes-it, thanks for the tip.
Any ETA for the real fix ?

**Comment by mdegat01** [comment]
Just got a bug report on this in supervisor. The `rtsp-to-web` addon's config.yaml lists a watchdog URL of `http://[HOST]:[PORT:8083]/`. But it's [config.yaml](https://github.com/allenporter/stream-addons/blob/main/rtsp-to-web/config.yaml) does not actually list any `ports` to expose. So it's creating an error where supervisor goes to look to see what port the user mapped port `8083` to and fails to find any ports.

**Comment by allenporter** [comment]
Thanks @mdegat01 really appreciate it.  Folks, i believe this is fixed with 1.2.2 so marking fixed though let me know if you still see issues.
